interview-questions.md

```diff
@@ -8,49 +8,56 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-  Your answer:
+  Your answer: Parameter is what you set call back for your variable data types.
+  Arguments reference the paramaters when you do your callbacks. Arguments must match the paramaters set in the variables
 
-  Researched answer:
+  Researched answer:Parameters are named variabless that get passed into a function and are used to import arguments into a function.
+  Argument are the values used to access the parameters passed into a function. 
 
 
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: .map() can take the index and the original array  as a needed parmater. Index is an optional parameter of .map().
 
+  Researched answer: .map() parametera are the definition of a function, Index, and callback of the original array. Index is an optional parameter, while function defition and callback of original array are needed parameters.
 
 
 3. What is the difference between map and filter?
 
-  Your answer:
-
-  Researched answer:
+  Your answer:.map() references each element on the array to return a new array in a function.
+  .filter() references an element that holds a certain characteristic(criteria) and can be used by a function without altering the original array.
 
+  Researched answer: map is a built in method that returns a new array with corresponding elements of the old array through a function. 
+  
+  filter does not change the original array but creates a new array with elements that pass a function test. It execute a function if the elements are empty.
 
 
 4. What is the difference between a function and a method?
 
-  Your answer:
+  Your answer:Function is code that you can re use multiple times until it meets a specifig task
+  Method is a function within an objec
 
-  Researched answer:
+  Researched answer:function is  a series of code block machines designed to perform a particular task or calculates a value. 
+  Method
```

Comment:
What was your researched answer on methods?

Reply:
Method is a function within an object